app/models/street/StreetEdgeTable.scala

```diff
@@ -17,6 +17,16 @@ import scala.slick.jdbc.{GetResult, StaticQuery => Q}
 
 case class StreetEdge(streetEdgeId: Int, geom: LineString, x1: Float, y1: Float, x2: Float, y2: Float, wayType: String, deleted: Boolean, timestamp: Option[Timestamp])
 
+case class OsmStreet(osmWayStreetEdgeId: Int, osmWayId: Int, streetEdgeId: Int)
+
+class OsmStreetTable(tag: slick.lifted.Tag) extends Table[OsmStreet](tag, Some("sidewalk"), "osm_way_street_edge") {
```

Comment:
Can you change the name to be `OsmWayStreetEdgeTable`? Our naming convention here has been to use the exact name of the table for this class.

Similarly, can you create a new file for it in the same directory as this file? We didn't really have a need for it in the past, but it seems more useful to make the file for it now :)